packages/regl-worldview/src/commands/GLTFScene.js

```diff
@@ -141,12 +142,15 @@ const drawModel = (regl) => {
       for (const primitive of mesh.primitives) {
         const material = model.json.materials[primitive.material];
         const texInfo = material.pbrMetallicRoughness.baseColorTexture;
+        if (!accessors || !textures) {
+          throw new Error("Error decoding GLB file");
```

Comment:
curious why include these changes in this PR? when did you run into this error?

Reply:
I didn't run into an issue but this was caught when fixing flow errors in this file. Adding this error allows us to guarantee `accessors` and `textures` are defined.